routers/routes/routes.go

```diff
@@ -710,14 +710,16 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Post("/purge", user.NotificationPurgePost)
 	}, reqSignIn)
 
-	m.Group("/api", func() {
-		apiv1.RegisterRoutes(m)
-	}, ignSignIn)
-
-	m.Group("/api/internal", func() {
-		// package name internal is ideal but Golang is not allowed, so we use private as package name.
-		private.RegisterRoutes(m)
-	})
+	if setting.API.EnableAPIEndpoint {
+		m.Group("/api", func() {
+			apiv1.RegisterRoutes(m)
+		}, ignSignIn)
+
+		m.Group("/api/internal", func() {
```

Comment:
`/api/internal` can not be disabled as this is called from ssh internally

Reply:
Fixed.